site/ohs_configuration.md

```diff
@@ -0,0 +1,49 @@
+## Configuring Oracle HTTP Server (OHS)
+
+Oracle HTTP Server (OHS) can be configured and updated with Create Domain, Update Domain, and Deploy Applications Tools, starting with WDT release 1.8.0. OHS configuration is only supported for offline mode in WDT. The OHS configuration can be discovered using the Discover Domain Tool, in offline mode only.
+ 
+### Prerequisites
+
+In order to configure and use OHS, the product must be installed in the Oracle Home used to create the domain. You can download OHS [here](https://www.oracle.com/middleware/technologies/webtier-downloads.html).
+
+The OHS template must be present in the WDT domain type definition file used to create or update the domain. See [Domain Type Definitions](type_def.md) for more information on creating a custom definition.
+
+You should create a copy of an existing domain type definition file, add the template to that file, then reference that file on the WDT command line. For example, if you want to create a domain with Oracle HTTP Server based on a Restricted JRF domain, then you would first create a copy of the file `WLSDEPLOY_HOME/lib/typedefs/RestrictedJRF.json` in the same directory, such as `WLSDEPLOY_HOME/lib/typedefs/HttpServer.json`. In this example, you would change the existing `extensionTemplates` section to include the additional OHS template. The original value is 
```

Comment:
You should create a copy -> You create a copy
file, then -> file, and then
The original value is -> The original value is: